go.mod

```diff
@@ -31,3 +39,5 @@ require (
 	k8s.io/klog v1.0.0
 	k8s.io/kubectl v0.17.3
 )
+
+replace github.com/docker/distribution => github.com/docker/distribution v0.0.0-20191216044856-a8371794149d
```

Comment:
is there any way to resolve this? we just got these all cleaned up...

Reply:
We could just not run an in-process docker registry, which I expect would make the tests more complex.

Reply:
This is also a test dependency, which I think means it's not required by dependents to use the operator-registry module.